Documentation/buildin block view

TomasArguelles opened the pull request:
Aunque arrasé con unas cuantas de las cosas de la plantilla lo hice basándome en este ejemplo 
[arc42byexample-sample.pdf](https://github.com/Arquisoft/dechat_es2b/files/3025626/arc42byexample-sample.pdf)


Comentad cosas a cambiar si veis alguna y si no dadle luz verde

TomasArguelles committed:
Add files via upload

añadido modelo actualizado a fecha 30/3/19

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Add files via upload

añadida un poco más curiosa

TomasArguelles committed:
Update 05_building_block_view.adoc

TomasArguelles committed:
Update 05_building_block_view.adoc

typos

acg96 commented:
En la imagen doc/raw/images/Building block view 1 2019-03-30 15-53-36.png pones vistas en español.
Yo seguiría el esquema de niveles en el que en el primer caso pondría la foto que indicas quitando lo de vistas, simplemente que interacciona con dechat (que no se porque hay dos archivos subidos cambiando la forma), luego pondría una interacción con los diferentes módulos de dentro de la app, en plan que llama a un servicio, el servicio al repositorio, lo que sea. Y explicaría como indica https://docs.arc42.org/section-5/ un poco las interfaces más importantes y demás. Luego haría otro nivel 2 en el que pondría y explicaría un poco más cada una de las cajas de antes (repos, servicios...)
El uml no lo pondría porque no es lo que se pide aquí o a mí no me encaja

TomasArguelles committed:
Delete Building block view 1 2019-03-30 15-53-36.png

borrada imagen repetida

TomasArguelles committed:
Delete Diagram 2019-03-30 15-53-36.png

esto no sé ni porque está aquí

TomasArguelles committed:
Add files via upload

quitado "(Vistas)"

TomasArguelles committed:
Delete Bulding Block 2019-03-30 15-56-58.png

TomasArguelles committed:
Delete Bulding Block.png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Delete Bulding Block.png

TomasArguelles committed:
Add files via upload

a ver si se sube bien

TomasArguelles committed:
Delete Bulding Block.png

TomasArguelles committed:
Add files via upload

TomasArguelles commented:
ignorad los commit de arriba, problemas con las imágenes

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Update 05_building_block_view.adoc

actualizado pero no completo, tengo que pensarlo más y dormir

enolgargon commented:
Yo veo les explicaciones muy breves. Y tengo una extraña obsesion por la que me duelen los ojos cuando veo archivos con espacios

TomasArguelles committed:
Delete Bulding Block eas.png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Update 05_building_block_view.adoc

Cambiada la imagen para reducir espacios en blanco, empezada de nuevo la parte de Whitebox Overall System.

TomasArguelles committed:
Delete Diagram 2019-04-01 09-29-13.png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Delete Diagram 2019-04-01 13-14-40.png

TomasArguelles committed:
Delete Diagram 2019-04-01.png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Delete Diagrama envio y lectura de mensajes (con service y repository).png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Delete Diagram 2019-04-01 13-28-52.png

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
Update 05_building_block_view.adoc

Update de todo hasta el Whitebox, mirad lo hecho a ver que opinais

TomasArguelles committed:
Update 05_building_block_view.adoc

No sé que más añadir para que los párrafos no sean tan cortos

enolgargon commented:
@acg96 puedes echar un vistazo? Yo lo veo aceptable

acg96 commented:
veolo todo bien menos en el  nivel 2 donde se desarrolla component, ahi deberian estar los componentes (messages, message, contact, contacts y alguno mas habia sino recuerdo mal)

TomasArguelles committed:
Add files via upload

TomasArguelles committed:
cambiada imagen de Component

cambiada imagen de Component y descripción para añadir message/s contact/s y module
